### DISCUSSION TITLE
SUGGESTION DOCUMENTATION VERSION

### DISCUSSION THREAD

**Discussion by LeonardoDaLuz** [saadeghi/daisyui#504] [Ideas / feature requests]
I've been using  the version 1.16.2 for about 3 months on a project. But the documentation is to version 2.1.0

Thanks for this UI framework! it's the best there is.

**Comment by saadeghi** [comment]
Thanks for the suggestion.  
The latest stable `v1` site is now available at https://v1.daisyui.com
